### ISSUE TITLE
KML Network link error when switching scene modes

### ISSUE THREAD

**Issue by hpinkos** [AnalyticalGraphicsInc/cesium#3865]
In this example, click the button to load the data.  Then after the data is loaded, switch the scene mode.  It causes a crash because camera.pitch is undefined when the scene mode is morphing.
This error only happens when you pass in `options.camera` and `options.canvas` (which is now required).

```
var viewer = new Cesium.Viewer('cesiumContainer');
var options = {
    camera : viewer.scene.camera,
    canvas : viewer.scene.canvas
};

Sandcastle.addToolbarButton('Load KML Network link',
    function() {
        var kmlText='<kml><NetworkLink>' +
            '<name><![CDATA[Test]]></name>' +
            '<refreshVisibility>0</refreshVisibility>' +
            '<flyToView>0</flyToView>' +
            '<Link><href><![CDATA[../../SampleData/kml/facilities/facilities.kml]]></href>' +
            '<viewRefreshMode>onStop</viewRefreshMode>' +
            '<viewRefreshTime>0</viewRefreshTime>' +
            '<viewFormat>BBOX=[bboxWest],[bboxSouth],[bboxEast],[bboxNorth]</viewFormat>' +
            '</Link></NetworkLink></kml>';
        var dom = new DOMParser().parseFromString(kmlText, 'text/xml');
        viewer.camera.flyHome(0);
        viewer.dataSources.add(Cesium.KmlDataSource.load(dom, options));
    });

Sandcastle.reset = function() {
    viewer.dataSources.removeAll();
    viewer.clock.clockRange = Cesium.ClockRange.UNBOUNDED;
    viewer.clock.clockStep = Cesium.ClockStep.SYSTEM_CLOCK;
};
```

**Comment by hpinkos** [comment]
Reported on the forum: https://groups.google.com/forum/?hl=en#!topic/cesium-dev/air2rOeFu8I

**Comment by hpinkos** [comment]
Closed by #3868
